app/assets/stylesheets/arclight/modules/hierarchy_and_online_contents.scss

```diff
@@ -141,6 +141,10 @@ $hierarchy-view-collapse-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3
 
   .online-contents {
     grid-area: online-contents;
+
+    .alert {
```

Comment:
This to me seems like we're using alert improperly.  These are cards, not alerts.  Can we remove the class from the template instead?